pkg/storage/version.go

```diff
@@ -1,11 +1,9 @@
 package storage
 
-import "io"
-
 // Version represents a version of a module and contains .mod file, a .info file and zip file of a specific version
 type Version struct {
 	Mod    []byte
-	Zip    io.ReadCloser
+	Zip    Zip
```

Comment:
This should probably be a pointer, because copying by value will still copy by reference the ReadCloser